<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargeAdjustmentDialog</class>
 <widget class="QDialog" name="ChargeAdjustmentDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>419</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Adjust Charges</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>381</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="groupCombo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0-69</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70-79</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80-89</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90-99</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="creditButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit</string>
   </property>
  </widget>
  <widget class="QLabel" name="groupLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Charge Group:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="buyerCommissionGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>290</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title">
    <string>Buyer Commission</string>
   </property>
   <widget class="QLabel" name="textLabel1_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="buyerCommAmount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Extension</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="buyerCommMode">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>CWT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Head</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Flat</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="textLabel1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Applies To:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="buyerCommExtension">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="payeeCombo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Buyer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Company</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="Q3Table" name="adjustmentTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>290</width>
     <height>180</height>
    </rect>
   </property>
   <property name="numRows">
    <number>0</number>
   </property>
   <property name="numCols">
    <number>2</number>
   </property>
   <column>
    <property name="text">
     <string>Charge Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Entry</string>
    </property>
   </column>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">types/invoice.h</include>
  <include location="local">types/commission.h</include>
  <include location="local">dao/chargedao.h</include>
  <include location="local">dao/invoicedao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
